contrib/dockerd-rootless-setuptool.sh

```diff
@@ -293,7 +293,9 @@ install_systemd() {
 			Documentation=https://docs.docker.com/go/rootless/
 
 			[Service]
-			Environment=PATH=$BIN:/sbin:/usr/sbin:$PATH
+			Environment="XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR"
```

Comment:
Curious; wouldn't this be inherited already from the environment?

Reply:
Yes, I don't get the point of this PR.